### PR TITLE
(mobile) Add meta theme-color

### LINKED ISSUE #1530: App theme color for mobile
It would be nice if there were a mobile theme-color for Chrome, probably #303030 to match the header.

### PR DESCRIPTION
**Pull request by HunterHarling**
Closes #1530

### CODE REVIEW

File: astrobin/templates/base/meta.html [hunk 1]
```diff
@@ -5,4 +5,4 @@
 <meta name="keywords" content="astrophotography, astronomy, telescopes" />
 <meta name="robots" content="index, follow, noarchive" />
 <meta name="googlebot" content="noarchive" />
-
+<meta name="theme-color" content="#303030" />
```
Comment (siovene) [inline astrobin/templates/base/meta.html:8]: Can you please do this also for the other base pages? 

- [ ] `image/full.html`
- [ ] `mainenance_base.html`
- [ ] `error_base.html`

Thanks!

**Review by siovene — CHANGES_REQUESTED** [review]

File: astrobin/templates/base/meta.html (hunk 1 above)
Comment (HunterHarling) [reply astrobin/templates/base/meta.html:8]: Yes, I had missed those. I added it to them now also.

**Comment by siovene** [comment]
Hi @HunterHarling , what am I supposed to see? Looks the same as without this, on my phone (Chrome for Android, Galaxy S10):

Screenshot_20201030-082211_Chrome
Screenshot_20201030-082222_Chrome

**Comment by HunterHarling** [comment]
That's interesting, it looks like your phone has a default gray color for the Chrome header. On my phone, the header is white by default, and different theme-colors change the header and top info bar to different colors.
Here is with theme-color of #303030 and then one of #CC4B2E:
Screenshot_20201030-013807
Screenshot_20201030-013819

**Comment by siovene** [comment]
@HunterHarling would you have time to do the same also for the Angular app?

**Pull request merged**